acceptance/bundle/resources/apps/script

```diff
@@ -0,0 +1,9 @@
+trace $CLI bundle deploy
```

Comment:
Is this test run on Cloud as well? If so, we need to destroy the bundle too

Reply:
No, it's local-only.